src/vs/workbench/contrib/chat/browser/chatInputPart.ts

```diff
@@ -981,6 +981,7 @@ export class ChatInputPart extends Disposable implements IHistoryNavigationWidge
 			supportIcons: true,
 			hoverDelegate,
 			title: localize('chat.attachment.clearButton', "Remove from context"),
+			ariaLabel: localize('chat.attachment.clearButtonAria', "Remove from context"),
```

Comment:
If they have the same ID they won't need different strings to get localized and result in slightly smaller bundle size

Reply:
oh, I thought that having the same ID was not allowed for localization